addon/search/search.js

```diff
@@ -63,12 +66,13 @@
       selectValueOnOpen: true,
       closeOnEnter: false,
       onClose: function() { clearSearch(cm); },
-      onKeyDown: onKeyDown
+      onKeyDown: onKeyDown,
+      bottom: cm.options.search.bottom
```

Comment:
This looks like it'll fail with "`cm.options.search` is undefined" when the option _wasn't_ provided.